pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -310,7 +314,8 @@ def preprocess_population_data(df_pop_raw: pd.DataFrame,
 def write_population_data(df_pop: pd.DataFrame,
                           out_folder: str = dd.defaultDict['out_folder'],
                           file_format: str = dd.defaultDict['file_format'],
-                          merge_eisenach: bool = True
+                          merge_eisenach: bool = True,
+                          ref_year='newest'
```

Comment:
think about another default value as mentioned above

Reply:
None seems reasonable to me